src/games/gameitem.py

```diff
@@ -285,9 +290,13 @@ def update(self, message, old_client=None):
         # Spawn announcers: IF we had a gamestate change, show replay and hosting announcements
         if oldstate != self.state:
             if self.state == "playing":  # The delay is there because we have a 5 minutes delay in the livereplay server
-                QtCore.QTimer.singleShot(5*60000, self.announceReplay)
+                self.launched_at = message['launched_at']
+                if time.time() - self.launched_at > 5*60:  # most already running games at client start
+                    self.announceReplay()
+                else:
+                    QtCore.QTimer().singleShot(5*60100 - 1000*(time.time() - self.launched_at), self.announceReplay)
             elif self.state == "open":  # The 3.5s delay is there because the host needs time to choose a map
-                QtCore.QTimer.singleShot(35000, self.announceHosting)
+                QtCore.QTimer().singleShot(35000, self.announceHosting)
```

Comment:
well, both versions seem to work. (I can't tell the difference)
Without the () PyCharm complain about the time number ('Expected type 'QTimer', got 'int' instead'), with the () no complain. So from PyCharm perspective with the () is better.
Then again, PyCharm isn't the wisest in Qt things.